content/400-pulse/200-getting-started.mdx

```diff
@@ -89,12 +89,16 @@ const prisma = new PrismaClient().$extends(
 )
 ```
 
+:::tip
```

Comment:
I think a warning may be more appropriate than a tip maybe?

Reply:
I agree that would be better 👍 !